### PR TITLE
Att/326 export profiles

### LINKED ISSUE #326: Export of Profile Visualizations
Use case arose where staff wanted to grab that are produced in community profiles for use in presentation (Saugus for example). Used screenshots, but would have benefited from easily downloaded high quality images.

### PR DESCRIPTION
**Pull request by atomtay**
Resolves #326.

# Why is this change necessary?
Internal staff use the Community Profiles section in some of their work, but didn't have a clear way to export the visualizations. The stopgap was taking screenshots of necessary charts and graphs.

# How does it address the issue?
Adding a "Print Visualizations" button makes it easy to either print or save the set of visualizations as a PDF. All charts are included by default, but through printer settings users can specify which pages they're interested in keeping. `break-before` and `break-after` styles are added to specify where the pages should break (between sections/tabs)

# What side effects does it have?
Every browser handles page-to-print slightly differently; below I'm attaching the results when running the charts for Palmer through Chrome, Safari, and Firefox. Chrome handles it best, while Firefox has some trouble with the initial page breaks and Safari adds in blank pages between each section (which seems to be a recurring issue, if you Google "safari prints blank pages") Most of our users (64% in the last month) use Chrome, so I'm not terribly concerned.

I also took this as an opportunity to re-factor some class-based components into functions and flatten some SCSS. Not directly related, but it made debugging the `@media print` queries a bit clearer.

[DataCommon-Firefox.pdf](https://github.com/MAPC/datacommon/files/5343044/DataCommon-Firefox.pdf)
[DataCommon-Safari.pdf](https://github.com/MAPC/datacommon/files/5343045/DataCommon-Safari.pdf)
[DataCommon-Chrome.pdf](https://github.com/MAPC/datacommon/files/5343043/DataCommon-Chrome.pdf)

### CODE REVIEW

**Review by mzagaja — APPROVED** [review]
This looks ready to merge. For the Safari issue I'm going to try to track down a relevant WebKit bug and/or file one.

**Comment by atomtay** [comment]
Ah shoot wait, in looking for an existing WebKit bug I came across [this StackOverflow post](https://stackoverflow.com/questions/7846346/how-to-avoid-extra-blank-page-at-end-while-printing/7847290) which reminded me that I was mixing both `break-before` and `break-after`; the blank pages were always in between sections, and I realized that Safari was accepting both the `break-after` rule from the last `tab__row`and the `break-before` from each `tab` section.

TLDR added a quick media rule for `tab__row--break` (not the best BEM name I've ever come up with but it gets the job done) so that only specified rows create a page break. This is important for the Environment section, which has 3 charts that don't quite all fit on a standard 8.5x11 page.

**Comment by mzagaja** [comment]
Much better, ship it!
